app/presenters/email_alert_signup_content_item_presenter.rb

```diff
@@ -0,0 +1,68 @@
+class EmailAlertSignupContentItemPresenter
+
+  def initialize(policy, update_type="major")
+    @policy = policy
+    @update_type = update_type
+  end
+
+  def exportable_attributes
+    {
+      format: "email_alert_signup",
+      content_id: policy.email_alert_signup_content_id,
+      title: policy.name,
+      description: description,
+      public_updated_at: policy.updated_at.iso8601,
```

Comment:
Is there a specific need to call `iso8601` here? That transformation should happen when the hash is transformed to JSON. For consistency with the other presenters, maybe best not to call it.